crates/proto/src/quic/quic_client_stream.rs

```diff
@@ -154,6 +154,21 @@ pub struct QuicClientStreamBuilder {
 }
 
 impl QuicClientStreamBuilder {
+    /// Creates a new QUIC connection builder.
+    pub fn new() -> Result<Self, ProtoError> {
+        let mut transport_config = quic_config::transport();
+        // clients never accept new bidirectional streams
+        transport_config.max_concurrent_bidi_streams(VarInt::from_u32(0));
+
+        let client_config = client_config_tls13()?;
+
+        Ok(Self {
+            crypto_config: client_config,
+            transport_config: Arc::new(transport_config),
+            bind_addr: None,
+        })
+    }
```

Comment:
If we don't want the builder to be fallible, we could fill the store at the end instead of the start.

Reply:
Done, apologies for the delay here!